odoo/addons/base/i18n/base.pot

```diff
@@ -4011,10 +4011,10 @@ msgid ""
 msgstr ""
 
 #. module: base
-#: model:ir.module.module,description:base.module_website_form_project
+#: model:ir.module.module,description:base.module_website_project
```

Comment:
Note that changes in `.pot` in master are useless (or at least not really needed) because they will be simply fully regenerated once we release the next version, see https://github.com/odoo/odoo/pull/113570#pullrequestreview-1313445887

But in this case, I guess it makes sense to not keep occurrences of `website_form_project` in the code, and we don't want to fully remove the `.pot` and `.po` files just for that.

So it's fine :+1:

Reply:
Ok thanks for the review. The branch has been rebased on community.